### ISSUE TITLE
Service config doesn't work !!!

### ISSUE THREAD

**Issue by Farrien** [editor-js/embed#68]
I did a lot of tests with config. And looks like it doesn't work.
It only supports default services from [official list](https://github.com/editor-js/embed/blob/master/docs/services.md).

A year ago: I added Twitch clips support but it can no longer work.

**Comment by Farrien** [comment]
I checked older editorjs version. So embed service configs works on [2.19.3](https://cdn.jsdelivr.net/npm/@editorjs/editorjs@2.19.3) or older

**Comment by guilhae** [comment]
Same for me! ☝️

**Comment by kaiz16** [comment]
Same here too. Had to downgrade to older version to work

**Comment by Farrien** [comment]
> Same here too. Had to downgrade to older version to work.

And devs do nothing)
What version do u use?

**Comment by kaiz16** [comment]
2.19.1

**Comment by jakubm95** [comment]
It doesn't work for me even with an older version of the editor.js
Even in the simplest configuration. `embed: Embed,`

**Comment by CiotkaCierpienia** [comment]
I've opened PR with the fix: https://github.com/editor-js/embed/pull/71, hope this will help at least with adding the plugin to editor

**Comment by dgastudio** [comment]
2.22.1
    ```
    embed: {
          class: Embed,
          config: {
            services: {
              youtube: true,
              vimeo: true,
              instagram: true,
            }
          }
        },


```

doesnt work  at all

**Comment by zhiln** [comment]
Same here! Custom embed service configs work fine for me on `@editorjs/editorjs` version 2.19.3 or older. Upgrading the version above 2.20.0 doesn't work. Any plans on fixing this bug?

**Comment by gohabereg** [comment]
Resolved in https://github.com/codex-team/editor.js/pull/1781, will be published with the next release

**Comment by abelikov5** [comment]
in my way, just update bundle.js by adding nessesary config 
![image](https://user-images.githubusercontent.com/48707335/136738419-09a94b6c-2ca7-498c-81a7-e65aecdbada9.png)

**Comment by sabigara** [comment]
In my environment, this is fixed in `@editorjs/editorjs@2.22.3`

**Comment by neSpecc** [comment]
Resolved
